src/couch_replicator/src/couch_replicator_worker.erl

```diff
@@ -349,8 +349,9 @@ bulk_get(#httpdb{} = Source, #{} = IdRevs) ->
     case couch_replicator_api_wrap:bulk_get(Source, IdRevs, Opts) of
         {ok, #{} = Docs} ->
             FilterFun = fun
-                (_, #doc{atts = []}) -> true;
+                (_, #doc{id = <<?DESIGN_DOC_PREFIX, _/binary>>}) -> false;
```

Comment:
I see minor optimization here - since we know right off the bat these are design documents, we don't have to fetch them in bulk_get at all (unlike the ones with attachments). So am thinking of updating to fetch and upload design docs, fetch the bulk docs, do the attachments ones. What do we think?

Reply:
Updated as described in the note